migrations/V31__active.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE company_survey
+    ADD COLUMN active BOOLEAN NOT NULL DEFAULT TRUE;
```

Comment:
as boolean the name could be more specific as "isActive" - "isReady" of "isIncludedInSimulation"

Reply:
changed to `include_in_simulation`